pkgs/by-name/ox/oxycards/package.nix

```diff
@@ -0,0 +1,29 @@
+{ lib
+, rustPlatform
+, fetchFromGitHub
+, unstableGitUpdater
+}:
+
+rustPlatform.buildRustPackage {
+  pname = "oxycards";
+  version = "unstable-2023-03-11";
+
+  src = fetchFromGitHub {
+    owner = "BrookJeynes";
+    repo = "oxycards";
+    rev = "aef415f17ee30b6af897121091912e190b70537d";
+    hash = "sha256-YNqNNW7sozuWppuRoM8dfx+HaIuLvqriq7V0KJJJkGI=";
+  };
+
+  cargoHash = "sha256-0mA5OAWL8pNP3riu2lBmmvMtYIBJH6eNX3Ibp5ikNj4=";
+
+  passthru.updateScript = unstableGitUpdater { };
+
+  meta = with lib; {
+    description = "Oxycards is a quiz card application built within the terminal";
+    homepage = "https://github.com/BrookJeynes/oxycards";
+    license = licenses.mit;
```

Comment:
Uh, there is no license upstream? This should be set to `licenses.unfree` in that case.

Reply:
**Thanks for the review!**

As for the License, I got the `MIT` license from the [Cargo.toml](https://github.com/BrookJeynes/oxycards/blob/main/Cargo.toml#L6).

Reply:
That is so weird lol. I guess this is fine then.

Reply:
I might open a PR upstream suggesting the addition of a `LICENSE` file.